pennylane/ops/qubit/parametrized_hamiltonian.py

```diff
@@ -0,0 +1,198 @@
+"""
+This submodule contains the time-dependent hamiltonian class
+"""
+import pennylane as qml
+from pennylane.operation import Observable, Tensor
+from pennylane.ops.qubit.hamiltonian import Hamiltonian
+
+
+# pylint: disable= too-many-instance-attributes
+class ParametrizedHamiltonian(Observable):
```

Comment:
Will `ParametrizedHamiltonian` be available at the top `qml` level of PennyLane? I'm not sure if we need it to be, since we won't typically expect users to directly use it (instead, it'd be via dunder methods or `dot`).

Reply:
Currently its available as `qml.ops.ParametrizedHamiltonian` but not directly via `qml`